src/components/profiles-modals/RenameProfileModal.vue

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+import { Vue, Component, Watch } from 'vue-property-decorator';
+import { ModalCard } from "../all";
+import sanitize from 'sanitize-filename';
+import R2Error from "../../model/errors/R2Error";
+import Profile from "../../model/Profile";
+
+@Component({
+    components: {ModalCard}
+})
+export default class RenameProfileModal extends Vue {
+    private newProfileName: string = '';
```

Comment:
This still has a bug, where the newprofilename is empty when the modal is first opened (for example coming from the game selection). Doing `private newProfileName: string = this.$store.state.profile.activeProfile.getProfileName();` doesn't work either. I'll look into this, but if you can immediately think of a solution feel free to comment.